buildSrc/build.gradle

```diff
@@ -112,6 +112,7 @@ dependencies {
   api 'commons-io:commons-io:2.7'
   api "net.java.dev.jna:jna:5.11.0"
   api 'gradle.plugin.com.github.johnrengelman:shadow:7.1.2'
+  api 'org.jdom:jdom2:2.0.6.1'
```

Comment:
Sorry, was (is) this one transitive before?

Reply:
Yes, jdom2 comes from `gradle.plugin.com.github.johnrengelman:shadow:7.1.2`, so making sure it picks the latest jdom2 version.